crates/next-binding/Cargo.toml

```diff
@@ -0,0 +1,132 @@
+[package]
+name = "next-binding"
+version = "0.1.0"
+edition = "2021"
+license = "MPL-2.0"
+autobenches = false
+
+[lib]
+bench = false
+
+[features]
+__swc = []
```

Comment:
To indicate this is not a public feature. This is a convention we used in upstream swc as well, similar to old javascript var naming to indicate it is not a public interface user can expect to use.

Reply:
But the whole package is not public, so this is just noise 🤷

Reply:
Yes true, but also technically next-swc is a consumer (in terms of visibility, not corp-wise relations) and this could be a sort of indication what's what. At least, these are literally internal features so we can change anytime if we don't feel this is not needed.